optaplanner-core/src/main/java/org/optaplanner/core/api/domain/variable/PlanningListVariable.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.core.api.domain.variable;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import java.util.List;
+
+import org.optaplanner.core.api.domain.entity.PlanningEntity;
+
+/**
+ * Specifies that a bean property (or a field) of a {@link List} type should be optimized by the optimization algorithms.
+ * Unlike {@link PlanningVariable}, the {@link PlanningListVariable} tells solver to change elements inside the list variable
```

Comment:
Not sure of this wording: "tells solver to change elements inside the list variable". It kinda make it look like a `@PlanningEntityCollection` property (telling OptaPlanner what entities to modify). Maybe something like "tells solver to change what elements are inside the list variable"

Reply:
Done.